specification/metadata/schema/providerversion.md

```diff
@@ -0,0 +1,30 @@
+# Provider Version
+
+The ProviderVersion MAY be provided to allow the provider to declare the version of logic by which the FOCUS dataset was generated and is separate from Focus Version. ProviderVersion allows for the provider to specify changes that may not result in a structural change in the data. It is suggested that the provider version use a versioning approach such as [SemVer](https://semver.org) version.
+
+## Metadata ID
+
+ProviderVersion
+
+## Metadata Name
+
+Provider Version
+
+## Requirements
+
+* The provider MUST document what changes are present in the ProviderVersion.
+* ProviderVersion MUST be of type String and MUST NOT contain null values.
```

Comment:
just sanity checking we intend this to be an incremental version number i.e. 1,2,3 or are we also allowing text values i.e. alpha, bravo, charlie

Reply:
We do not specify, we do suggest a versioning format such as SemVer below.